src/components/MeetupList.js

```diff
@@ -5,6 +5,12 @@ import Emoji from "./Emoji"
 import Link from "./Link"
 
 const meetups = [
+  {
+    title: "DeFi Dubai",
+    emoji: ":uae:",
+    location: "Dubai",
+    link: "https://www.meetup.com/DeFi-Dubai/",
+  },
```

Comment:
Emoji is broke. I _think_ this should fix it?

```suggestion
  {
    title: "DeFi Dubai",
    emoji: ":united_arab_emirates:",
    location: "Dubai",
    link: "https://www.meetup.com/DeFi-Dubai/",
  },
```